pyproject.toml

```diff
@@ -6,8 +6,8 @@ requires = [
 	 "setuptools<60.0",  # Do not increase, 60.0 enables vendored distutils
 	 "Cython>=0.29.21",
 	 "python-dev-tools",
-	 "oldest-supported-numpy; python_version>='3.9'",
-	 "numpy; python_version<'3.9'"
+	 "oldest-supported-numpy; python_version>'3.9'",
+	 "numpy; python_version<='3.9'"
```

Comment:
Would it be helpful for you if I did `"numpy>='1.20'; python_version<='3.9'"?

Reply:
Do you mean `"numpy=='1.20'; python_version<='3.9'"`? or such a pin would cause problems? (if so, then just leaving it as `"numpy; python_version<='3.9'"` is fine)

(I suggested 1.19 before because of `"numpy>=1.19.5"` in the dependencies).

To be clear, this bug is specifically about supporting older numpy versions. Without a lower-or-equal numpy version in the build dependencies as what is installed in the environment, then the install doesn't work right. It could just be left as currently suggested and add `pip install apexpy --no-build-isolation --no-cache` as the install instruction for some cases.

Reply:
I expanded the text around the conda failure section to more fully describe what's going on.  You can check it out here:  https://github.com/aburrell/apexpy/pull/103/files#diff-d9b149498982c0663c3b7170398773361ed5678f1a627e9c2fd8d2c955c563dbR190

Reply:
Makes sense! thanks